mautic/mautic
# Limit number of (custom) field

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | X
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Required: )
#### Description:
If too many (custom) field is creating. The fields table exceed the maximum of add up column size.
So the field is creating in lead_field table but not in fields table, this bug create many another bug.
Ex1: You can't use this field in all Mautic.
⚠️ Ex2: Doctrine try to drop some field of leads table in leads when you use doctrine:schema:update ⚠️ 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Create many custom field (~200 items)
2. See in database, some last field is create in lead_field but not in leads or you can see in the page to add a custom field , you are not redirect you and get a error 500 when you submit the form.

#### Steps to test this PR:
1. Apply this PR
2. Create many custom field (~200 items)
3. See in database, any field created in lead_field and in leads, if is not possible. Or you can see a error flashbag when you submit a custom field.
